servers/rendering/renderer_rd/storage_rd/texture_storage.cpp

```diff
@@ -2102,6 +2114,17 @@ void TextureStorage::_update_render_target(RenderTarget *rt) {
 
 	Vector<RID> fb_textures;
 	fb_textures.push_back(rt->color);
+
+	if (rt->msaa != RS::VIEWPORT_MSAA_DISABLED) {
+		RD::TextureView rd_view_resolve;
+		// Reuse the texture format of the color attachment.
+		rd_format.samples = RD::TEXTURE_SAMPLES_1;
```

Comment:
i am not understanding, why are you overriding the number of samples from L2106? Given this looks intentional, can oyu be a little more clear on the comment in the previous line?

Reply:
I reuse rd_format since it only functions as a configuration object. The color_resolve buffer should have the same color format/size/etc. but it must have a sample count of 1 since the multisampled color buffer is resolved later into color_resolve. Adjusted the comment.